docs/source/migrating.rst

```diff
@@ -5,7 +5,7 @@
 Migrating from IPython
 ======================
 
-`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+`The Big Split <http://blog.jupyter.org/2015/04/15/the-big-split/>`__
```

Comment:
@minrk I suspect that you are correct that it is an OpenSSL issue with OS X. 
Python 3.5
OpenSSL 0.9.8zg 14 July 2015
OS X 10.10.5